dali/test/python/autoserialize_test/custom_module_inside.py

```diff
@@ -0,0 +1,9 @@
+from nvidia.dali.plugin.triton import autoserialize
+from nvidia.dali import pipeline_def
+
+
+@autoserialize
+@pipeline_def(batch_size=1, num_threads=1, device_id=0)
+def func_under_test():
+    import numpy as np  # noqa: F401
```

Comment:
It is likely that the test env may have numpy installed. How about `import doesnt_exist`?